ngraph_bridge/enable_variable_ops/ngraph_catalog.h

```diff
@@ -38,6 +38,12 @@ namespace ngraph_bridge {
 
 class NGraphCatalog {
  private:
+#if (NGRAPH_TF_USE_GRAPPLER_OPTIMIZER)
+  // Map of tensorflow variable name that has been replaced to a NGV to shared
+  // name
+  static unordered_map<string, string> tf_var_name_to_shared_name_map_;
```

Comment:
Discuss: Perhaps we do not need a map of var name -> shared_name
All we need is a set of varnames
(because the current variable under consideration can tell us its shared name (assuming shared names do not change))